scrapy/utils/project.py

```diff
@@ -68,18 +66,10 @@ def get_project_settings():
     if settings_module_path:
         settings.setmodule(settings_module_path, priority='project')
 
-    # XXX: remove this hack
-    pickled_settings = os.environ.get("SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE")
-    if pickled_settings:
-        warnings.warn("Use of environment variable "
-                      "'SCRAPY_PICKLED_SETTINGS_TO_OVERRIDE' "
-                      "is deprecated.", ScrapyDeprecationWarning)
-        settings.setdict(pickle.loads(pickled_settings), priority='project')
-
-    # XXX: deprecate and remove this functionality
+    # XXX: deprecating the block below
     env_overrides = {k[7:]: v for k, v in os.environ.items() if
                      k.startswith('SCRAPY_')}
     if env_overrides:
+        warnings.warn("Use of environment variable 'SCRAPY_' is deprecated.", ScrapyDeprecationWarning)
```

Comment:
I'd rephrase this a bit to clearly indicate the purpose of the variables. Something like `Use of 'SCRAPY_'-prefixed environment variables to override settings is deprecated`

Reply:
Would `usage` be better here? I'm not a native speaker, I read a few comments about use/usage but I'm still not sure.